src/lib/EDSL.mli

```diff
@@ -92,6 +92,35 @@ val string : string -> string t
 val int : int -> int t
 val bool : bool -> bool t
 
+module Integer : sig
+  val to_string : int t -> string t
+  val of_string : string t -> int t
+  val bin_op : int t -> [ `Div | `Minus | `Mult | `Plus | `Mod ] -> int t -> int t
+  val add : int t -> int t -> int t
```

Comment:
I'm curious why you want to expose this function? Is there a standard way in shell to represent this logic specifically, or said differently, is this a shell idiom?